libs/designsystem/src/lib/components/chart/chart.component.ts

```diff
@@ -1,12 +1,15 @@
 import { Component, OnInit, Input, OnChanges, ElementRef, Inject } from '@angular/core';
-import { Options, PlotSeriesDataLabelsOptions } from 'highcharts';
+import { Options, PlotSeriesDataLabelsOptions, XAxisBreaksOptions, XAxisOptions } from 'highcharts';
 
 import { ChartHelper } from './chart-helper';
 import { ChartType } from './chart-type';
 import { DONUT_OPTIONS, DonutOptions } from './options/donut';
 import { AREASPLINE_OPTIONS, AreaSplineOptions } from './options/areaspline';
 import { TIMESERIES_OPTIONS, TimeSeriesOptions } from './options/timeseries';
 import { ACTIVITYGAUGE_OPTIONS, ActivityGaugeOptions } from './options/activitygauge';
+import { mergeDeep } from '../../helpers/deep-merge';
+import { getColumnOptions } from './options/column';
```

Comment:
Why a method when the other default options are constants?

Reply:
So we can pass values to the configuration